### ISSUE TITLE
Mobile scrolling bug

### ISSUE THREAD

**Issue by maxwoollons** [codergautam/worldguessr#19]
When scrolling on mobile the map scrolls down as if there is content below. 

https://youtube.com/shorts/bDXKbU0Iru8?si=YNvhMTlSdSIZPDSY

**Comment by maxwoollons** [comment]
I will have a look into this and see if there's anything I can do.

**Comment by maxwoollons** [comment]
This also includes the scrolling of the country flags when you go into the menu

**Comment by codergautam** [comment]
Yes this is an iphone specific bug, I unfortunately don't have an iphone so I couldn't test much on this, if you are able to fix this that would be AMAZING many people told me about this but idk whats causing it (doesnt do it on android)

**Comment by janardannn** [comment]
assign me

**Comment by codergautam** [comment]
Let me know if you need any help investigating this

**Comment by janardannn** [comment]
@m626zNq did it solve the problem? did you test it ?

**Comment by m626zNq** [comment]
> @m626zNq did it solve the problem? did you test it ?

Waiting for response from @maxwoollons , I have an android and I don't have the problem so.

**Comment by janardannn** [comment]
will do

**Comment by m626zNq** [comment]
@janardannn @maxwoollons

**Comment by janardannn** [comment]
its not showing scrollbar in ios 18, iPhone 13

**Comment by codergautam** [comment]
Great!
